### ISSUE TITLE
Skin 10.6.0-0: Focus rings are missing from ebay-menu-button-item when user clicks on individual item

### ISSUE THREAD

**Issue by ushrivastava123** [eBay/skin#1177]
@imcburnie
when I navigate through ebay-menu-button-item via keyboard, the focus rings appear correctly
however when I click through individual item, focus rings are missing.
Markup:
```
 <div id="menu_basic" class="grid__cell grid__cell--4of16 grid__cell--4of16-lg grid__cell--4of8-sm" draggable="true" ondragstart="drag(event)">
      <h3>Basic</h3>
      <ebay-menu-button text="eBay Menu" on-menu-button-select(console.log.bind(console, "menu-sel evectent is triggered")) on-menu-button-collapse(console.log.bind(console, "menu-collapse evectent is triggered"))>
        <ebay-menu-button-item>item 1</ebay-menu-button-item>
        <ebay-menu-button-item>item 2</ebay-menu-button-item>
        <ebay-menu-button-item>item 3</ebay-menu-button-item>
      </ebay-menu-button>
    </div>
```
Buggy (10.6.0-0)
![image](https://user-images.githubusercontent.com/57548118/87357849-94d40a80-c519-11ea-97dc-9e89887ade07.png)

Non-buggy (10.5.0-0)
![image](https://user-images.githubusercontent.com/57548118/87357917-af0de880-c519-11ea-865e-f1de4def77be.png)
